src/tools/wasmopcodecnt.cc

```diff
@@ -354,21 +354,27 @@ int main(int argc, char** argv) {
     init_opcnt_data(&opcnt_data);
     result = read_binary_opcnt(data, size, &s_read_binary_options, &opcnt_data);
     if (WABT_SUCCEEDED(result)) {
+      fprintf(stderr, "here1\n");
```

Comment:
I assume these fprintf's are for debugging? Should they be removed (Or at least protected via condition, or hidden behind a macro)?